tests/utils/pvc.go

```diff
@@ -0,0 +1,87 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+
+	k8sv1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	DefaultPvcMountPath = "/pvc"
+	defaultPollInterval = 2 * time.Second
```

Comment:
I was ambiguous in my comment lat night. I meant to suggest that you include in the poll const _what_ is being polled. Eg `pvcPollInterval` so we know the resource this applies to is a pvc. The _utils_ pkg may eventually have many different resources that are polled and it nice to have an interval specific to each. Though, in fairness, I commonly see `2s` used as the interval in many unrelated polls.

Reply:
The same argument applies to the timeout as the poll value. So I will change it.